### PR TITLE
Icarus

### PR DESCRIPTION
**Pull request by mkosunen**
@Roenski @Anttitarkka If you approve this, we are done with Icarus. I fixed the latency problem by sampling the output at falling clock.

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -9,14 +9,13 @@ cat << EOF > ./TheSDK.config
...
-echo 'LSFSUBMISSION="bsub -K"'
-echo 'LSFINTERACTIVE="bsub -I"'
-fi)
+#if [ -z "`command -v bsub`" ]; then
+#echo '#LSFSUBMISSION="bsub -K"'
+#echo '#LSFINTERACTIVE="bsub -I"'
+#else
+#echo 'LSFSUBMISSION="bsub -K"'
+#echo 'LSFINTERACTIVE="bsub -I"'
+#fi)
```
Comment (Roenski) [inline configure]: I believe these should be uncommented

Comment (mkosunen) [reply configure]: Solved by
 b52565374e66df27191b0a4

**Comment by mkosunen** [comment]
@Roenski @Anttitarkka If you have no further comments, 'icarus' branches  can be merged deleted.

**Comment by Roenski** [comment]
Related changes have been applied to v1.9_RC branch.

**Pull request closed without merging**